### ISSUE TITLE
Comments on the proposal draft

### ISSUE THREAD

**Issue by botwithfeelings** [RobotWithFeelings/CSC710_Project#1]
@meneal, the proposal draft looks really good and close to the final submission. Nonetheless, here are my thoughts on the proposal draft. Feel free to let me know which of these seem worthy of being included in the draft.
## First two paragraphs:
- One point I learned so far from the reading groups is that every paragraph should focus on a single goal statement. The first paragraph here is to focus on Nass' study and it's results. The second sentence seems to point to the results but it's not explicitly mentioned (last part of the second sentence - in this way).
- The subject population demographic of the study is not required here I think. The second paragraph should focus on why it is worthwhile to do Nass' study again now as the access to computers has become more common. 

See my changes in ShareLatex for the first two paragraphs, feel free to change them as you see fit. I have your changes in block comments untouched.
## Proposed study:

The part involving our study roadmap should have the following goals:
- What we will do.
- What is our expected results.
- What are the challenges of performing the study, if any.

I made some changes in SharLatex according to this, check them to see if you like these. We can further discuss on the changes on github or in person.

**Comment by meneal** [comment]
The changes so far I think are awesome.  I made a few grammatical changes
throughout.  We have personalization throughout the paper in a weird way
though I think.  Should that not be personification?

I also shortened the title.  Seemed to me that should be enough, what do
you think?

Overall I think the changes are great and what you've put up here for
requirements for proposed study and first two paragraphs are really helpful
to me.
